### PR TITLE
Create side navigation for logged in pages

### PR DESCRIPTION
**Pull request by antimega**

### CODE REVIEW

**Comment by antimega** [comment]
Added sidebar navigation to logged in pages:

screen shot 2015-12-14 at 16 58 45

Added a template for the navigation - main_nav.html and a template for logged in pages - withnav_template.html .

Removed visited colour from links.

I want to make the nav smaller but I'll do that in another pull request.

File: app/templates/main_nav.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{% macro main_nav() %}
+
+<div id="navigation">
+<h3><a href="/dashboard">Service name</a></h3>
+<ul>
+	<li><a href="/sms/send">Send text messages</a></li>
```
Comment (quis) [inline app/templates/main_nav.html]: These URLs should use [Flask’s `url_for()`](http://flask.pocoo.org/docs/0.10/quickstart/#url-building). You can copy most of them from: https://github.com/alphagov/notifications-admin/pull/30/files#diff-cbedc13d9eac3fe1b43fd45e5935ffc5L14

**Comment by antimega** [comment]
Fixed now, thanks @quis.

**Comment by quis** [comment]
Looks good now, :+1:

**Pull request merged**